src/helpers/textFormat.ts

```diff
@@ -0,0 +1,14 @@
+function spacePadString(numString: string) {
+    if (numString.length === 1) {
+        return '  ' + numString
+    } else {
+        return numString
+    }
+}
```

Comment:
`spacePadString` doesn't really explain what the function is doing. How many spaces are we padding it by? Why are we padding the string?